problems/service-placement/problem-statement-sp.md

```diff
@@ -0,0 +1,146 @@
+<!--
+SPDX-FileCopyrightText: 2024 Noé Godinho <noe@dei.uc.pt>
+
+SPDX-License-Identifier: CC-BY-4.0
+-->
+
+# Service Placement Problem
+
+Noé Godinho, University of Coimbra, Centre for Informatics and Systems of the University of Coimbra, Department of Informatics Engineering.
+
+Copyright 2024 ... Noé Godinho.
+
+This document is licensed under CC-BY-4.0.
+
+## Introduction
+
+One of the main problems in communications is the routing problem, where the goal is to find a path between two devices that minimises a given metric. Usually, that metric is latency or number of hops. However, that problem is too restrictive, due to bandwidth or device usage not being taken into account when solving the problem.
+
+The service placement problem was proposed to tackle these issues. The goal is to optimise a set of requests in a network, where each request is composed by source and destination, bandwidth requirements and a set of services to offload, with CPU and RAM requirements. This problem is NP-complete, making it hard to solve in realistic scenarios. Thus, efficient techniques to obtain (near-)optimal solutions are necessary. 
+
+## Task
+
+Given a graph, a set of requests and their services to be offloaded, minimise the overall latency and processing time of all requests and services, taking resource constraints into account.
+
+## Detailed description
+
+Given **N** nodes, **E** edges, **R** requests and **S$_r$** services for each request $r$, the following objective function aims to minimise latency and processing time:

```

Comment:
```suggestion
Given $N$ nodes, $E$ edges, $R$ requests and $S_r$ services for each request $r$, the following objective function aims to minimise latency and processing time:
```
For notation consistency it would be better to use inline math here.